skins/laika/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "axios": "^0.18.0",
     "buefy": "^0.7.3",
+    "bulma-checkradio": "^1.1.0",
```

Comment:
Why do we need this package?
I checked the docs and it literally says
>  Make classic checkbox and radio sexier with different colors, sizes, and states

 :see_no_evil:

Reply:
We need a custom radio button design because all browsers render radio buttons differently. This bulma package makes them looks very similar to Jan's design so I would just use it rather than building something with cross-browser functionality ourselves.

Reply:
But `Buefy` already has [radio buttons ](https://buefy.org/documentation/radio/)

Reply:
@tzhelyazkova Thanks, you're right, I had totally forgotten about the extra design elements that Buefy adds. I'll check it out.